trade-finance-manager-ui/server/controllers/case/activity/index.test.js

```diff
@@ -18,354 +18,341 @@ const session = {
+      });
 
-    it('should redirect to not-found route', async () => {
-      const req = {
-        params: {
-          _id: '1',
-        },
-        session,
-      };
+      it('should redirect to not-found route', async () => {
+        const req = {
+          params: {
+            _id: '1',
+          },
+          session,
+        };
 
-      await activityController.getCommentBox(req, res);
-      expect(res.redirect).toHaveBeenCalledWith('/not-found');
+        await activityController.getCommentBox(req, res);
+        expect(res.redirect).toHaveBeenCalledWith('/not-found');
+      });
     });
   });
-});
 
-describe('POST activity - post-comment', () => {
-  const mockDeal = {
-    _id: '61f6ac5b02fade01b1e8efef',
-    dealSnapshot: {
+  describe('POST activity - post-comment', () => {
+    const mockDeal = {
       _id: '61f6ac5b02fade01b1e8efef',
-      submissionDetails: {
-        supplierName: 'test supplier',
+      dealSnapshot: {
+        _id: '61f6ac5b02fade01b1e8efef',
+        submissionDetails: {
+          supplierName: 'test supplier',
+        },
       },
-    },
-    tfm: {
-      activities: [],
-    },
-  };
-
-  const longComment = 'aaaaa'.repeat(201);
-  const normalComment = 'test';
-
-  describe('when deal exists', () => {
-    beforeEach(() => {
-      api.getDeal = () => Promise.resolve(mockDeal);
-      api.getAmendmentsByDealId = () => Promise.resolve({ data: [] });
-    });
+      tfm: {
+        activities: [],
+      },
+    };
 
-    it('should return to comment box with errors if longer than 1000 characters', async () => {
-      const req = {
-        params: {
-          _id: mockDeal._id,
-        },
-        body: {
-          comment: longComment,
-        },
-        session,
-      };
+    const longComment = 'aaaaa'.repeat(201);
+    const normalComment = 'test';
 
-      await activityController.postComment(req, res);
-      expect(res.render).toHaveBeenCalledWith('case/activity/activity-comment.njk', {
-        dealId: mockDeal.dealSnapshot._id,
-        user: session.user,
-        maxCommentLength: 1000,
-        validationErrors: generateValidationErrors('comment', 'Comments must be 1000 characters or fewer', 0, {}),
-        comment: longComment,
+    describe('when deal exists', () => {
+      beforeEach(() => {
+        api.getDeal = () => Promise.resolve(mockDeal);
+        api.getAmendmentsByDealId = () => Promise.resolve({ data: [] });
       });
-    });
 
-    it('should return render activities page with all-activity filter after posting comment', async () => {
-      const req = {
-        params: {
-          _id: mockDeal._id,
-        },
-        body: {
-          comment: normalComment,
-        },
-        session,
-      };
-      const activities = mapActivities(mockDeal.tfm.activities);
-
-      await activityController.postComment(req, res);
-      expect(res.render).toHaveBeenCalledWith('case/activity/activity.njk', {
```

Comment:
These were only passing because the mocks weren't being cleared. This controller should never have called res.render